VolleyPaint/Assets/Scripts/Player/Shooting/PlayerShoot.cs

```diff
@@ -188,6 +227,5 @@ private bool IsBallShot()
     public void ReplenishAmmoClientRpc()
```

Comment:
hmm i dont think other players need to know about when u replenish ammo so u shouldn't need an RPC for this

Reply:
This is old code that isn't going to be used, I will clean that up.